### PR TITLE
feat: add a template for noop jobs

### PR DESCRIPTION
**Pull request by jithine**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: sd/noop
+version: '1.0.0'
+description: SDv4 template that does nothing
+maintainer: jithin1987@gmail.com
+config:
+  image: screwdrivercd/noop-container
+  steps:
+    - nop: "true"
```
Comment (scr-oath) [inline template.yaml]: Should the step be "noop" ?

**Review by scr-oath — APPROVED** [review]
Make the step be `noop` to agree with the image

**Review by scr-oath — CHANGES_REQUESTED** [review]
`noop`

File: template.yaml (hunk 1 above)
Comment (jithine) [reply template.yaml]: updated

**Review by scr-oath — APPROVED** [review]

**Pull request merged**